tests/Helpers/Tokens/AsymmetricVerifierTest.php

```diff
@@ -111,8 +111,9 @@ public static function getTokenBuilder() : Builder
         return (new Builder())->withClaim('sub', '__test_sub__')->withHeader('kid', '__test_kid__');
     }
 
-    public static function getToken(string $rsa_private_key, Builder $builder = null) : Token
+    public static function getToken(string $rsa_private_key = null, Builder $builder = null) : Token
     {
+        $rsa_private_key = $rsa_private_key ?? self::getRsaKeys()['private'];
```

Comment:
is this default something that can be moved to the method signature? or does that not accept this static invocation inline?